lib/demos/cupertino/cupertino_picker_demo.dart

```diff
@@ -63,13 +63,15 @@ class _CupertinoPickerDemoState extends State<CupertinoPickerDemo> {
             ),
           );
         },
-        child: _Menu(children: [
-          Text(GalleryLocalizations.of(context)!.demoCupertinoPickerDate),
-          Text(
-            DateFormat.yMMMMd().format(date),
-            style: const TextStyle(color: CupertinoColors.inactiveGray),
-          ),
-        ]),
+        child: _Menu(
+          children: [
+            Text(GalleryLocalizations.of(context)!.demoCupertinoPickerDate),
+            Text(
+              DateFormat.yMMMMd().format(date),
+              style: const TextStyle(color: CupertinoColors.inactiveGray),
+            ),
+          ],
+        ),
```

Comment:
Is this just a change in formatting (indentation)?

Reply:
exactly